### DISCUSSION TITLE
how to get dropfiles in webview?

### DISCUSSION THREAD

**Discussion by wuyouqiang** [MicrosoftEdge/WebView2Feedback#4113] [Q&A] [answered]
When I drag multiple files to the webview, only one of the files will be opened, and the remaining files will not be opened.

now we use the  bellow code :

 hr = webview_->add_NewWindowRequested(Callback<ICoreWebView2NewWindowRequestedEventHandler>([this, &browser](ICoreWebView2* sender, ICoreWebView2NewWindowRequestedEventArgs* args) -> HRESULT {
        BOOL handled = FALSE;
        wil::unique_cotaskmem_string url;

        HRESULT hr = args->get_Uri(&url); // but it get the one drop file.
        if (!SUCCEEDED(hr))
            return hr;
}

How to get multiple dragged and dropped file lists in webview?

**Comment by victorhuangwq** [comment]
@johna-ms could you help this user out?

**Comment by johna-ms** [comment, marked as answer]
@wuyouqiang I believe this scenario is unsupported by normal edge browser. You cannot drag and drop multi-selected files into browser. If it is not supported there, we are unlikely to add support for it in webview. Please correct me if I am wrong.

**Reply by wuyouqiang** [reply]
thanks , got it .
